configs/experiment/bool_mask_audio_jepa_with_dataset.yaml

```
@@ -0,0 +1,14 @@
+# @package _global_
+
+defaults:
+  - bool_mask_audio_jepa
+  - override /interaction/environment: audio_dataset
+
+interaction:
+  environment:
+    observation_generator:
+      audio_files: ${audio_data_dir}
```

やっぱりここは直接指定したいな。何を使って学習する想定なのか明確にしたい気持ち
データのパスは ホストOSの `/mnt/Archive/ami-logs/random_observation_action_log.20241123`にある。
```suggestion
      audio_files:
        - random_observation_action_log.20241123/2024-11-10_11-00-15/io/audio_recordings/2024-11-10_11-00-15.858323.wav
        - ...
```